internal/logstore/lokistack/logstore_lokistack.go

```diff
@@ -129,9 +138,9 @@ func newLokiStackWriterClusterRoleBinding() *rbacv1.ClusterRoleBinding {
 	)
 }
 
-func newLokiStackAppReaderClusterRole() *rbacv1.ClusterRole {
+func newLokiStackAppViewClusterRole() *rbacv1.ClusterRole {
```

Comment:
There are three functions only differing in two small places now. Should we replace this with a function that takes two parameters?

Reply:
What impact are you referring to? As far as I can see the functions are only used in one place.